### ISSUE TITLE
First instance of Code opens full-screen, subsequent instances open in smaller windows

### ISSUE THREAD

**Issue by DanTup** [Microsoft/vscode#15294]
- VSCode Version: 1.7.1
- OS Version: Windows 10

In a command window if I run `code .` it opens code fullscreen (which I want; it's how I left it). If I then run `code .` again from another command window, it opens in a small window (not full screen).

This is 100% reproducible for me. It's incredibly minor, but it is a little annoying (it's an extra step to maximise every time I do it) and it does feel a bit wonky.

**Comment by bpasero** [comment]
This issue has already been reported. Please use the search function next time before reporting an issue and/or this time if you are curious about the progress of the issue.

**Comment by DanTup** [comment]
@bpasero Apologies if I've raised a dupe, but I really did search and couldn't find a case that seemed to be this. I just tried again, searching for `fullscreen`, `full-screen`, `full screen`, `maximised`, `maximized` and even `window` but none of the results seem to be the issue I reported (I haven't read through every result but I have read the titles for the first several pages (`window` had 180 pages!) and clicked through to the ones that seemed like they could be this).

I don't want to take up your time but if you know what the issue might be titled and are able to provide the issue number so I can subscribe, I would appreciate it.
